lib/streams/documentStream.js

```diff
@@ -28,7 +28,22 @@ function createDocumentStream(id_prefix, stats) {
         if (record.POSTCODE) {
           addrDoc.setAddress( 'zip', record.POSTCODE );
         }
-
+        
+        if (useOaAdminFields) {
+          if (record.CITY) {
+            addrDoc.parent.locality.push(record.CITY);
```

Comment:
Please use `addrDoc.addParent('locality', record.CITY)` which ensures valid data instead of adding values to the arrays directly.  See https://github.com/pelias/whosonfirst/blob/master/src/peliasDocGenerators.js#L58